

try {
    console.log('Work')
    console.log(a)
    conaole.log('Processing')
} catch (error) {
    console.log(error)
}

console.log('done')